.rubocop.yml

```diff
@@ -162,6 +162,8 @@ Style/ConditionalAssignment:
   Enabled: false
 Style/EmptyMethod:
   EnforcedStyle: expanded
+Style/FetchEnvVar:
+  Enabled: false
```

Comment:
Is there an option to only disable the nil case?

Reply:
not sure, it seems like the cop is not active if the `ENV['']` style is used in `||` chains though, see https://github.com/rubocop/rubocop/pull/10819